packs/drone/variables.hcl

```diff
@@ -0,0 +1,145 @@
+variable "job_name" {
+  description = "The name to use as the job name which overrides using the pack name."
+  type        = string
+  default     = ""
+}
+
+variable "datacenters" {
+  description = "A list of datacenters in the region which are eligible for job placement."
+  type        = list(string)
+  default     = ["dc1"]
+}
+
+variable "region" {
+  description = "The region where the job should be placed."
+  type        = string
+  default     = "global"
+}
+
+variable "namespace" {
+  description = "The namespace where the job should be placed."
+  type        = string
+  default     = "default"
+}
+
+variable "constraints" {
+  description = "Constraints to apply to the entire job."
+  type        = list(object({
+    attribute = string
+    operator  = string
+    value     = string
+  }))
+  default = [
+    {
+      attribute = "$${attr.kernel.name}",
+      value     = "linux",
+      operator  = "",
+    },
+  ]
+}
+
+variable "group_network" {
+  description = "The Drone network configuration options."
+  type        = object({
+    mode  = string
+    ports = map(number)
+  })
+  default = {
+    mode  = "bridge",
+    ports = {
+      "drone-server" = 80,
+      "drone-agent"  = 3000,
+    },
+  }
+}
+
+variable "drone_server_image" {
+  description = "The Drone Server image."
+  type        = string
+  default     = "drone/drone"
+}
+
+variable "drone_server_version" {
+  description = "The Drone Server image version."
+  type        = string
+  default     = "2.1.0"
+}
+
+variable "drone_server_cfg" {
+  description = "The Drone Server configuration."
+  type        = string
+  default     = <<EOF
+DRONE_USER_CREATE=username:admin,machine:false,admin:true
```

Comment:
If this doesn't have the following vars, the task dies:

DRONE_GITHUB_CLIENT_ID=<ID>
DRONE_GITHUB_CLIENT_SECRET=<SECRET>

I think we should probably add them to this and the drone_agent_cfg below, and then add them to the readme as well.

Reply:
Ah, correction, you need these or one of every other env vars.

I can update the ReadMe with a link to their docs to give future users a hint on where to go.